echoshader/new_version/echogram.py

```diff
@@ -17,33 +29,74 @@ def __init__(self, MVBS_ds):
             "Image": {
                 "cmap": "jet",
                 "colorbar": True,
-                "cmap": "jet",
                 "tools": ["box_select", "lasso_select", "hover"],
                 "invert_yaxis": False,
                 "width": 600,
-            }
+            },
+            "RGB": {"tools": ["hover"], "invert_yaxis": False, "width": 600},
         }
 
-    def echogram(
+        self._init_widget()
+
+    def _init_widget(self):
+        """
+        Initialize the interactive widgets for controlling the echogram visualization.
+        """
+        self.channel_select = panel.widgets.Select(
+            name="Channel Select", options=self.MVBS_ds.channel.values.tolist()
+        )
+
+        self.Sv_range_slider = panel.widgets.EditableRangeSlider(
+            name="Sv Range Slider",
+            start=self.MVBS_ds.Sv.actual_range[0],
+            end=self.MVBS_ds.Sv.actual_range[-1],
+            value=(self.MVBS_ds.Sv.actual_range[0], self.MVBS_ds.Sv.actual_range[-1]),
+            step=0.01,
+        )
+
+        self.color_map = panel.widgets.TextInput(
+            name="Color Map", value="jet", placeholder="jet"
+        )
+
+    def echograms(
```

Comment:
to make it more obvious when comparing this with `echogram` which is just for single frequency/channel
```suggestion
    def echogram_multiple(
```